src/portals/blockchain-frontend/apps/blockchain-frontend/pages/certificates/index.tsx

```diff
@@ -115,10 +116,6 @@ export function CartPageCertificates() {
 
   return (
     <div>
-      {contextHolder}
```

Comment:
this is needed for showing the toast message

Reply:
Fixed